### ISSUE TITLE
I'm new to this, but I think I followed the instructions correctly.

### ISSUE THREAD

**Issue by TheSuckySix** [Yimura/YimMenu#2642]
### Describe the build issues

So, I'm new to this, ever since my ISP started blocking access to simpler menus somehow I've had to get progressively more creative in finding something that works. I followed the instructions, but there was a bit of a deviation in that I had to install winget through powershell because I couldn't get it working through its own installer. I don't think that affects anything, because ultimately winget is on my pc, right where it should be. I then installed visual, cmake, all the good stuff, built the thing, got the injector, and I can successfully inject the menu. It loads, until it stops. The first time, I got an error prompt message about something being corrupted. I rebuilt the project, stopped getting that specific error message, but still got crashes. That was the second time. Third time, I uninstalled the entirety of yimmenu, reinstalled it using CMD same as before, re-re-built the project, still crashes the same way as the second time.

### Screenshots and/or Logs

[cout.log](https://github.com/YimMenu/YimMenu/files/13760424/cout.log)

### IDE/Compiler

Visual Studio 2022

### Additional context

For some inexplicable reason, my ISP has been blocking access to Crespomods, Amnesia Menu, Scooby, Terror, MrModzz, *everything.* Due to the increasingly complicated situation, I went to github, a place my ISP cannot reasonably block access to. I had two primary features I wanted. Spawn/Duplicate cars, customize them, gift them to other players. That's what I'm here for.

### Setup Guide

- [X] I confirm that I followed the setup guide correctly.

### YimMenu Source Code

- [X] I've confirmed this is an issue with building unmodified YimMenu.

**Comment by gir489returns** [comment]
According to the log nothing is wrong and YimMenu is functioning correctly. Can you simplify your life story down to a few sentences of what you think is wrong?

**Comment by Yimura** [comment]
Could you maybe make a video of the issue occurring as the explanation doesn't really give much in what could've gone wrong.

**Comment by TheSuckySix** [comment]
[cout.log](https://github.com/YimMenu/YimMenu/files/13760447/cout.log)
![image](https://github.com/YimMenu/YimMenu/assets/154710704/4ee8c227-1c39-4bcd-98cb-497a4de1e6fe)

https://github.com/YimMenu/YimMenu/assets/154710704/9fd59709-0b2d-46a2-bcad-7da14ca8981d
Here's images, the log, and the video.

**Comment by gir489returns** [comment]
You're compiling in Debug mode. Compile in Release mode and try again.

**Comment by TheSuckySix** [comment]
Oh, thank you! It works! I appreciate the lightning fast support by the way, you guys respond like you're paid to do it. 

When I compiled the program, even the very first time, cmake automatically did it in debug mode. 

I looked up how to make it compile in release mode, maybe consider adding to the instructions a mention you'll need to make sure it's set to compile in release mode, just to further idiot-proof the process.
